common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -801,6 +804,49 @@ public static UTF8String concatWs(UTF8String separator, UTF8String... inputs) {
     return res;
   }
 
+  /**
+   * Return a locale of the given language and country, or a default locale when failures occur.
+   */
+  private Locale getLocale(UTF8String language, UTF8String country) {
+    try {
+      return new Locale(language.toString(), country.toString());
+    } finally {
+      return Locale.getDefault();
+    }
+  }
+
+  /**
+   * Splits a string into arrays of sentences, where each sentence is an array of words.
+   */
+  public ArrayList<ArrayList<UTF8String>> sentences(UTF8String language, UTF8String country) {
```

Comment:
Oh, I thought it's a convention.

+1. Splitting not-native UTF8String functions sounds good to me. We need to split the followings together.

- split
https://github.com/apache/spark/blob/master/common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java#L795

- translate
https://github.com/apache/spark/blob/master/common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java#L805

- toTitleCaseSlow
https://github.com/apache/spark/blob/master/common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java#L408